src/Core/Geographic/Extent.js

```diff
@@ -528,6 +537,16 @@ class Extent {
      * @return {Extent}
      */
     static fromBox3(crs, box) {
```

Comment:
Could you change the doc to match the method name ? It should probably take a `THREE.Box3` as a parameter.